org.tuxfamily.hatari.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.tuxfamily.hatari
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: hatari
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=all
```

Comment:
Any specific reason allow `--device=all`

Reply:
I believe this is is required to allow the use of joysticks and game controllers - correct me if I'm wrong

Reply:
Make sense.